### DISCUSSION TITLE
How to work with multiple domains

### DISCUSSION THREAD

**Discussion by juliengloria** [jasontaylordev/CleanArchitecture#1123] [Q&A]
Hi everyone,
I try to find a structure project with multiple domains. 
For exemple I could have many databases with their own migrations, different models ...
If someone has a repo to show this, it would be perfect  :)

**Comment by jasontaylordev** [comment]
I recommend exploring the implementation of Domain-Driven Design (DDD) with Clean Architecture, particularly focusing on the [bounded context](https://martinfowler.com/bliki/BoundedContext.html) pattern. This pattern is a crucial concept in DDD that helps structure projects by dividing larger domains into smaller sub-domains, such as Orders, Products, and Customers. By adopting this approach, each sub-domain can have its own models, database schema, and migrations, leading to a more modular and maintainable architecture.
